lib/external_logger/send_log.rb

```
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'external_logger/base'
+require 'forwardable'
+
+class ExternalLogger::SendLog < ExternalLogger::Base
+  class << self
+    extend Forwardable
+
+    def_delegators :logger, :info, :debug, :error, :fatal
+
+    def warn(message)
+      self::ADDITIONAL_LOGGERS.each do |name, active|
```

Should we raise an error here if someone adds a new logger to ADDITIONAL_LOGGERS but forgets to send the write method for this logger?